sidekick/models.py

```diff
@@ -9,6 +9,8 @@ class Organization(models.Model):
     url = models.CharField(max_length=200, null=False, blank=False)
     token = models.CharField(max_length=200, null=False, blank=False)
     users = models.ManyToManyField(User, related_name="org_users")
+    engage_url = models.CharField(max_length=200, null=True)
```

Comment:
I'm wondering whether we want to use a [`urlfield`](https://docs.djangoproject.com/en/2.1/ref/models/fields/#urlfield) here.